crates/protocol/src/registry.rs

```diff
@@ -161,6 +161,11 @@ impl PackageId {
     pub fn name(&self) -> &str {
         &self.id[self.colon + 1..]
     }
+
+    /// Gets the full string package identifier.
+    pub fn id(&self) -> &str {
+        &self.id
+    }
```

Comment:
Do we need this in addition to the `AsRef<str>`?

Reply:
Nope, I missed that. I'll remove.